### ISSUE TITLE
softmax: Shapes are not in standard layout / SSD

### ISSUE THREAD

**Issue by cagery** [ROCmSoftwarePlatform/AMDMIGraphX#728]
ONNX file:
https://github.com/onnx/models/blob/master/vision/object_detection_segmentation/ssd/model/ssd-10.onnx

When parsed with cpp_parse_load_save application, the error is:
```
./parse_load_save /cgMIGraphX/AMDMIGraphX_v1/examples/cpp_parse_load_save//models/vision/object_detection_segmentation/ssd/model/ssd-10.onnx --parse onnx
Parsing ONNX File
MIGraphX Error: /cgMIGraphX/AMDMIGraphX/src/include/migraphx/check_shapes.hpp:136: standard: softmax: Shapes are not in standard layout
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to call function
Aborted (core dumped)

```

**Comment by scxiao** [comment]
The issue is fixed in PR #740

**Comment by mvermeulen** [comment]
Fixed.
